spec/arrays.dd

```diff
@@ -229,7 +229,7 @@ writeln(b[7]);      // 10
 
 $(H2 $(LNAME2 array-copying, Array Copying))
 
-        $(P When the slice operator appears as the left-hand side of an
+        $(P When the slice appears as the left-hand side of an
```

Comment:
The paragraph explains the difference between:
```D
int[] a, b;
b[] = a; // slice operator, copy elements of a to b
b = a; // slice without operator, make b refer to the same data as a
```
Removing 'operator' is not correct here, the `[]` is important.

Reply:
Yes, my proposed wording change may not be the best.  But is there a **slice operator** in D?  Isn't `[]` the array subscript operator in D just like it is in C and C++? So  the actual operators in the expression

```d
b[] = a; // slice operator, copy elements of a to b
```
are the  `=` and `[]` operators? Where is the **slice operator** in the expression?  I don't know precisely how to improve the comment, but maybe simply saying

```d
b[] = a; // copies elements of a to b
```
is clearer than mentioning a non-extant **slice operator**. I'm new to D and coming from C++ but the _only_ **slice operator** that I am aware of is the tilde (~) operator described in the article [D Slices](https://dlang.org/articles/d-array-article.html) used for concatenation and appending.

Reply:
The section [Operator Overloading](https://dlang.org/spec/operatoroverloading.html#slice_unary_operators) of the  [Operator Overloading](https://dlang.org/spec/operatoroverloading.html) page of the Language Reference does explain that the  unary `[]` operator is overloaded for slices.

Obviously `[]` is the index operator for arrays and slices, but this simple explanation seems to be missing from the documentation. The only explanation of the unary `[]` operator of the D languange seems to be [Index Expressions](https://dlang.org/spec/expression.html#IndexExpression), and it is a very technical explanation.

Reply:
'Slice operator' may be poorly defined in the spec.

> Where is the slice operator in the expression? 

The `=` is the operator, and the `[]` on the left hand side makes `=` a slice operator. You can change the wording or clear it up with a better explanation, but removing 'operator' from that sentence without anything else makes it worse.

Reply:
Yes, I agree that simply removing 'operator' is not an improvement.

Reply:
Are you going to change it?

Reply:
No.